utils/interfaces/_agent.py

```diff
@@ -143,6 +143,22 @@ def get_spans(self, request=None):
     def get_spans_list(self, request):
         return [span for _, span in self.get_spans(request)]
 
+    def get_metrics(self):
+        """Attempts to fetch the spans the agent will submit to the backend.
```

Comment:
Noticed that the comment says `spans` instead of `metrics` in some places.

Reply:
Oops, thanks for spotting this. Fixed now!